config/fedora.yml

```diff
@@ -0,0 +1,4 @@
+user: fedoraAdmin
```

Comment:
I think ActiveFedora is hard-coded to load its own config file, unfortunately? That means we need at least a fedora.yml and a solr.yml and whatever the silly file names are for the test versions of those.